.travis.yml

```diff
@@ -2,6 +2,8 @@ language: ruby
 addons:
   postgresql: "9.3"
   firefox: "84.0.2"
+node_js:
```

Comment:
this whole file is going away.  You may want to coordinate with @aterrype about setting up this step in the github CI action she's working on.

Reply:
I'm planning to merge my PR in today, since it's approved now. I think it makes more sense to add node to the new CI steps in this PR rather than mine, because without the rest of this PR we aren't doing anything with node so it's harder to be sure that the setup is working.

Reply:
Just merged my PR (and updated the project CI rule).

Reply:
Added node/yarn to the CI pipeline. Looks like its working correctly.